pom.xml

```diff
@@ -111,8 +111,8 @@
     <scalacheck.version>1.12.5</scalacheck.version>
 
     <!-- frontend maven plugin related versions-->
-    <node.version>v12.3.1</node.version>
-    <npm.version>6.9.0</npm.version>
+    <node.version>v14.17.0</node.version>
```

Comment:
why choose an EOL version?

https://endoflife.date/nodejs

Reply:
If so, why not stay at node 12 to minimize the scope of this PR?

Reply:
To avoid trouble, I updated to the oldest Node version that is still supported by npm 9. However, this should actually be done in a separate PR. I removed the npm update.